### PR TITLE
Add Toast styles for Darkly theme

### LINKED ISSUE #943: Missing Toast styles
I've noticed that, at least, the Darkly theme does not support/adjust Bootstrap's [Toast](https://getbootstrap.com/docs/4.4/components/toasts/) component.

The Toast is displayed in light colors, although every other component is dark (e.g. Modal, Card). Due to transparency the Toast content is barely visible. Furthermore the close button is not visible at all, because it uses the `.close` class which applies the _correct_ text color white.

Screenshot 2020-02-06 at 12 52 47

The Toast component, as any other component, [uses variables](https://github.com/twbs/bootstrap/blob/master/scss/_toasts.scss), so that changing the appearance should be straightforward. I don't know whether custom adjustments have to be made, though.

## Environment
* Bootstrap v4.4
* Bootswatch v4.4.1 Darkly theme

### PR DESCRIPTION
**Pull request by HarmvZ**
*Preview*
image

Closes #943

### CODE REVIEW

**Comment by thomaspark** [comment]
Looks great, thanks for the PR @HarmvZ!

File: dist/darkly/_variables.scss [hunk 1]
```diff
@@ -147,6 +147,11 @@ $modal-content-border-color:        $gray-700 !default;
 
 $modal-header-border-color:         $gray-700 !default;
 
+// Toasts
+$toast-background-color:            $gray-700 !default;
+
```
Comment (jdoubleu) [inline dist/darkly/_variables.scss]: I've also added `$toast-border-color: $gray-700 !default;`.

The Card component uses the `$gray-700` color for its border while Bootstraps' default border color for Toasts is `rgba(0, 0, 0, .1)` (see https://github.com/twbs/bootstrap/blob/v4.4.1/scss/_variables.scss#L914).

**Comment by jdoubleu** [comment]
Thanks for addressing the issue!

As mentioned in the issue description, other themes might be affected, too. I can at least think of Cyborg which is also dark.

The Toast component should also be shown on the demo page of each theme.

If that's too much to ask, I am happy to create another PR and try to resolve it myself.

**Comment by thomaspark** [comment]
@jdoubleu, toasts should be fixed now for dark themes (and Sketchy). Let me know if you run into any other issues with them!

**Comment by jdoubleu** [comment]
Wow, that was fast. Thank you!

**Comment by HarmvZ** [comment]
@thomaspark Could you create a new release for this? I would like to use the updated version as an NPM package.

**Comment by thomaspark** [comment]
@HarmvZ Unfortunately we'll have to wait for the next Bootstrap version to be able to bump the version number for npm.

**Comment by jdoubleu** [comment]
According to the [`v4.4.2` Bootstrap project](https://github.com/twbs/bootstrap/projects/20) there is not much to do for the upcoming release. The last release happened on [Nov, 28th](https://blog.getbootstrap.com/2019/11/28/bootstrap-4-4-1/).

However I don't know their release cycle.

I'm not sure about just updating to the `master` version.

**Pull request merged**